CMakeLists.txt

```diff
@@ -111,10 +112,9 @@ endif ()
 
 
 # Set the default namespace
-if (OSL_NAMESPACE)
-    add_definitions ("-DOSL_NAMESPACE=${OSL_NAMESPACE}")
-endif ()
-message(STATUS "Setting Namespace to: ${OSL_NAMESPACE}")
+set (PROJ_NAMESPACE "${${PROJ_NAME}_NAMESPACE}")
```

Comment:
Ok no problem, it did dawn on me that the namespace config change would be a bit of a breaking back port change.
I'll try a build with HIDE_SYMBOLS turned on and see if this works for us.
Cheers